### ISSUE TITLE
[Feature Request] Hide nodes

### ISSUE THREAD

**Issue by Wasenob** [Openarl/PathOfBuilding#409]
I was trying to theory craft a build from scratch and realized that I was continuously re-reading the same passive nodes on my path. 

It would be useful to me, and hopefully others, if their was some way to "mark" or "hide" passive nodes that I knew I really didn't plan to use (like melee nodes for a bow build). 

If you wanted to take it a step further, someway to mark certain node types could be very helpful as well, such as highlighting bow nodes, or HP nodes, etc. Like when you use the search feature, but more permanent.

**Comment by Openarl** [comment]
Are you aware of the Node Power feature? It visually highlights nodes that improve your offence or defence, which is pretty much what you're asking for, only it doesn't require any user input other than simply turning it on (there's a checkbox at the bottom right of the Tree tab called "Show Node Power").

**Comment by Wasenob** [comment]
Yes, I use node power too. And it's very useful in it's own way, but not every node is about DPS or EHP. It doesn't help too much when looking for utility nodes, and some things just don't add to sheet numbers by virtue of game mechanics (Point Blank) or how the tool functions (bleed chance). 

I just though it would be very helpful to be able to scan the tree and say "These are the nodes I want to consider, and these are the ones I have no interest in for this build."

Keep up the great work, your tool is amazing!

**Comment by Openarl** [comment]
You do have a point about utility nodes. I will consider this; there's quite a few improvements for the passive tree that I'd like to make once I find the time, but the way things are going at the moment it may be a while before that happens.

**Comment by Mizari** [comment]
Hello! Wanted to create the same issue, is this in plans by any chance? ;d For example: shift-click will hide the node and all the nodes, that are behind it.

**Comment by lucom** [comment]
Hello! I was wondering whether there was any news regarding this feature request. I too have the same issue: When planning a new character I constantly re-read certain nodes. It would be great to simply mark them via CTRL click. That way this feature could not only be used to not re-read certain nodes all the time but also to plan the tree ahead. A button next to the search that clears all marks would be great and another button to mark all search results.

For example: Search for "minions", click on "mark search". Search for "evasion", click on "mark search". CTRL click a few other nodes, etc. Now I can either focus on all minion and evasion related nodes or I can ignore them more easily.

**Comment by Hypasist** [comment]
I also support this request. In my opinion, NodePower feature is not sufficient, when damage output of more than one skill is needed to be taken under consideration.
Ability to simply hide given node, whole tree branch or even node categories from tree would simplify theory-crafting process .

**Comment by mauritsunkel** [comment]
Hi, I've been thinking about this feature for years, I really hoped it would've been here now! Endorsing this request <3
